js-src/upload/index.ts

```diff
@@ -87,10 +90,14 @@ const upload: () => void = () => {
                 })
                 .catch(err => {
                     console.log(err);
-                    const error: HTMLParagraphElement | null = document.querySelector(
-                        ".error"
+                    const error: HTMLDivElement | null = document.querySelector(
+                        ".govuk-error-summary"
+                    );
+                    const displayError: HTMLParagraphElement | null = document.querySelector(
+                        ".errorMessage"
                     );
-                    error!.innerText = "There has been an error";
+                    displayError!.innerText = err;
+                    error!.style.display = "block";
```

Comment:
As with the inline style above, this should add or remove a CSS class rather than a direct call to `element.style`. But it's probably fine for now, unless the dynamic styles start to get confusing.

Reply:
Resolved